src/components/restorePassword/restorePassword.tsx

```diff
@@ -25,6 +25,8 @@ export type FormPass = {
 
 type Alert = 'success' | 'error';
 
+const ALERT_SUCCESS = 'success';
```

Comment:
I think it could be moved to general constant file, as you have duplication, I don't see why duplication needed

Reply:
I put on the constants/ links.ts because on .env the google auth don't work, looks like the function don't read the .env